pkg/storage/s3/s3.go

```diff
@@ -96,15 +99,18 @@ func (d *driver) createAndTagBucket(svc *s3.S3, clusterID string) error {
 		return err
 	}
 
+	d.Config.ManagedStorage = true
+
+	var tagSet []*s3.Tag
+	tagSet = append(tagSet, &s3.Tag{Key: aws.String("tectonicClusterID"), Value: aws.String(installConfig.ClusterID)})
+	for k, v := range installConfig.Platform.AWS.UserTags {
+		tagSet = append(tagSet, &s3.Tag{Key: aws.String(k), Value: aws.String(v)})
+	}
```

Comment:
add a test that confirms this is being set?  or do we not have usertags for the clusters being stood up by CI?

Reply:
Currently the UserTags are empty, so there is nothing to check.

Reply:
ok

Reply:
>  or do we not have usertags for the clusters being stood up by CI?

CI sets `expirationDate`, openshift/release#1761.

Reply:
I added testing that checks the UserTags against the tags we set on the s3 bucket